src/graphics/program-lib/chunks/fogExp.frag

```diff
@@ -1,9 +1,9 @@
 uniform vec3 fog_color;
 uniform float fog_density;
-
+float d_fog_factor=1.0; //for additive blend, this should be 0.0
```

Comment:
I would just call this `dFogFactor`. Isn't it just uniforms (and attributes) that follow the underscore naming convention?

Reply:
renamed to follow the naming convention correctly

Reply:
(I think) - I left in the 'd' - because its used like other 'd' globals

Reply:
hmm - maybe dFogBlendFactor could be better? Or dFogBlendTypeFactor?

Reply:
You went with `dFogFactor` which is fine by me.

Reply:
I changed it again - its now dBlendModeFogFactor - I'm thinking this makes it a bit more useful as it will appear when  searching for code related blend modes or fog?

Reply:
Looks good.